weeks/week02/HW2_Task3.py

```diff
@@ -0,0 +1,34 @@
+# List of steps:
+steps_1 = ["open browser", "navigate to page", "click login"]
+steps_2 = ["open browser", "navigate to page", "fill form"]
+steps_3 = ["open browser", "navigate to page", "click login"]  # duplicate steps
+
+# Use frozenset() to store the test steps of each scenario in an immutable set:
+scenario_1 = frozenset(steps_1)
+scenario_2 = frozenset(steps_2)
+scenario_3 = frozenset(steps_3)
+
+# Store each test scenario in a dictionary where the key is the test case name and the value is the frozenset of steps:
+dic_scenarios = {"Test Case 1": scenario_1, "Test Case 2": scenario_2, "Test Case 3": scenario_3, }
+
+new_steps = ["open browser", "navigate to page", "click login"]
+
+
+# new_steps = "open browser", "navigate to page", "fill form"
```

Comment:
It is a bad practice to keep not required comments

Reply:
will remove, thank you!